lib/internal/bootstrap_node.js

```diff
@@ -309,21 +309,19 @@
   }
 
   function setupInspectorCommandLineAPI() {
-    const inspector = process.binding('inspector');
-    const addCommandLineAPIMethod = inspector.addCommandLineAPIMethod;
-    if (!addCommandLineAPIMethod) return;
+    const { addCommandLineAPI } = process.binding('inspector');
+    if (!addCommandLineAPI) return;
 
     const Module = NativeModule.require('module');
+    const { makeRequireFunction } = NativeModule.require('internal/module');
     const path = NativeModule.require('path');
     const cwd = tryGetCwd(path);
 
-    const consoleAPIModule = new Module('[consoleAPI]');
-    consoleAPIModule.filename = path.join(cwd, consoleAPIModule.id);
-    consoleAPIModule.paths = Module._nodeModulePaths(cwd);
+    const consoleAPIModule = new Module('<inspector console>');
+    consoleAPIModule.paths =
+      Module._nodeModulePaths(cwd).concat(Module.globalPaths);
```

Comment:
lib/buffer.js consistently uses 2-space indentation (grep for `=$`), but either way it's fine for me.